apps/src/client.rs

```diff
@@ -446,3 +553,16 @@ pub fn connect(
 
     Ok(())
 }
+
+/// Generate a new pair of Source Connection ID and reset token.
+fn generate_cid_and_reset_token<T: SecureRandom>(
```

Comment:
There are two of these and they seem identical?

If so, please put common code in apps/src/common.rs

Reply:
Good point, moved the duplicate codes into common.rs